docs/ref-manual/cvl/ghosts.md

```diff
@@ -2,9 +2,117 @@
 Ghosts
 ======
 
+Ghosts are a way of defining global variables for use during verification.
```

Comment:
not sure about global. how about "additional"  or "Auxiliary"

Reply:
Changed to additional